cmd/manager/main.go

```diff
@@ -389,6 +400,12 @@ func startOperator(stopChan <-chan struct{}) error {
 		return err
 	}
 
+	// set the timeout for API client
+	cfg.Timeout = viper.GetDuration(operator.KubeClientTimeout)
+
+	// set the timeout for Elasticsearch requests
+	esclient.DefaultESClientTimeout = viper.GetDuration(operator.ElasticsearchClientTimeout)
```

Comment:
For consistency the default observer timeout & interval could also be configurable through flags?

Reply:
I feel like that'll just clutter the already long list of configuration options. Under normal circumstances, observer timeout and frequency are not things that users need to be concerned with. They only need to change those settings in exceptional circumstances, so leaving those to annotations feels right to me. WDYT?

Reply:
I think I'm fine with that.
My gut feeling says the observer timeout should match the es client timeout though (whose value can be specified in the annotation). So basically the observer timeout setting could disappear, I think?
We can open a different issue to better refine those.

Reply:
I tried to stick to what we already had in the code without changing the behaviour too much. The observer is sort of a health check if you squint hard enough :smile: So, it's probably fine to be more aggressive than normal. Of course, we can discuss it later in a different issue if you feel differently.